canonical-web-and-design/ubuntu.com
# Move /openstack/training to /training

## Done

- Move /openstack/training to /training
- Add redirect for /openstack/training to /training
- Remove `Training` from the secondary Openstack nav
- Add `Transfer and training` link in the main nav Kubernetes section

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- [x] Go to [/training](https://ubuntu-com-canonical-web-and-design-pr-5372.run.demo.haus/training) and make sure the content is the same as https://ubuntu.com/openstack/training
- [x] Go to [/openstack/training](https://ubuntu-com-canonical-web-and-design-pr-5372.run.demo.haus/openstack/training) and check if you are redirected to [/training](https://ubuntu-com-canonical-web-and-design-pr-5372.run.demo.haus/training)
- [x] Click on the `Enterprise` menu at the top and check if the `Transfer and training` link is in the `Kubernetes` section


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1294

## Screenshots

[if relevant, include a screenshot]
